### PR TITLE
Install license when available

### PR DESCRIPTION
**Pull request by brycepg**
#4

### CODE REVIEW

**Comment by wenLiangcan** [comment]
@brycepg Thanks for your contribution.     

I tried your code and encountered a problem when generating the install license statement for the `Flask` module.       

The generated statement is:
```shell
install -D -m644 artworkLICENSE "${pkgdir}/usr/share/licenses/python-flask/LICENSE"
```    

and the archive contains these license files:
```
LICENSE
artwork/LICENSE
```    

I know it is hard to implement a smart enough generation algorithm to fit all situations, so, I think it is better to make this feature optional and should be enabled explicitly by the command line argument.       

Also, I would prefer to keep the filename of the license file original rather than force renaming it to `LICENSE`.

**Comment by brycepg** [comment]
Good catch. I tried a few libraries out, but assumed that the compression library would list the files in some type of breadth first search instead of depth first search. Is there any interest in adding unit tests to this package?

I'll add a cli-arg like `--find-license`, and change the heuristics to prefer the license with the lowest directory depth.

**Comment by wenLiangcan** [comment]
Yes, I have plan to write some unit tests, but I'm afraid I have no time to do this in recent weeks.

File: pip2pkgbuild/pip2pkgbuild.py [hunk 1]
```diff
@@ -27,6 +36,8 @@
 MODULE_JSON = 'http://pypi.python.org/pypi/{name}/json'
 VERSION_MODULE_JSON = 'http://pypi.python.org/pypi/{name}/{version}/json'
 
+MAINTINER_LINE="# Maintainer: {name} <{email}>\n"
```
Comment (wenLiangcan) [inline pip2pkgbuild/pip2pkgbuild.py]: @brycepg A little coding style issue at this line, one space should be added to both sides of the assignment operator.

**Comment by brycepg** [comment]
@wenLiangcan I update my pull request to reflect your requests. The find license attempt will only be made when the `-l` or `--find-license` option is added, and it prefers licenses closer to the root so that your flask example doesn't pull the artwork license.

Additionally, I preserved the license name when installing and fixed some of the formatting.

**Comment by wenLiangcan** [comment]
Thanks @brycepg , merged.

**Pull request merged**